### ISSUE TITLE
Move "Wage Requests" to policy

### ISSUE THREAD

**Issue by Matroftt** [CorsixTH/CorsixTH#2649]
### What feature or enhancement would you like to add?

Move "Wage Requests" from Options to Policy tab and provide it with a more better explanation.

### How would this feature benefit the gameplay of CorsixTH?

Make it a more understandable feature

### Was the feature in the original game?

No

### What version of CorsixTH are you using?

0.68 rc-2

### Does this feature/enhancement relate to a current problem?

No

### Additional Information

![image](https://github.com/user-attachments/assets/34239c49-2570-4a37-9861-28f40175bede)
Make it look similar to this

**Comment by lewri** [comment]
Given the way we generate this window it should be doable. However, wage request is a config option so to really implement this you'd have to drop it from the config.txt file.

**Comment by inferno986return** [comment]
I like this. The wording could be more concise and understandable though maybe "Automatically grant wage requests?"

**Comment by ARGAMX** [comment]
> I like this. The wording could be more concise and understandable though maybe "Automatically grant wage requests?"

Sorry, I don't think that "Automatically grant wage requests" is more understandable.
Because for me it means that wage requests will be handled automatically for "Yes" value.
But this is not true, because a fax will arrive first anyway. So player can handle it manually for "Yes" and "No" cases.

**Comment by Matroftt** [comment]
> Edit: Not to say this wouldn't work if you still had it as a config option, but it certainly wouldn't make logical sense if it lived here.

I agree that it wouldnt make logical sense, but I would go with keeping it in the config. In my opinion, it can be annoying to select this option to "Yes" every new game if it will be "No" by default.
